packages/rum-core/test/common/utils.spec.js

```diff
@@ -291,4 +291,15 @@ describe('lib/utils', function() {
     )
     expect(utils.removeInvalidChars('invalid"')).toEqual('invalid_')
   })
+
+  it('should remove sensitive auth from url', () => {
+    expect(utils.removeAuthFromUrl('https://a:b@c.com/d')).toEqual(
+      'https://c.com/d'
+    )
+    expect(utils.removeAuthFromUrl('http://e:f@example.com/g')).toEqual(
```

Comment:
perhaps in this one you could put "e:f@" in the path, to show that only the userinfo is removed?

Reply:
Is it common to have user info in the path? We already remove the query params which might have sensitive info..

Reply:
No sorry if I was unclear, my point was that if there's by coincidence the same string in the path, it shouldn't be removed. That's a weird/pathological case, a more realistic one would be that there's a `<some string>@<some other string>` in the path that does not match the userinfo, which should not be modified.

I see you're handling that in your regex, I just think it's a good idea to have a test to make sure that doesn't break in the future.

Reply:
Got it now. thanks for the explanation 👍 Will add a test for the scenario.